mayastor/src/core/descriptor.rs

```diff
@@ -37,6 +37,10 @@ impl Descriptor {
     pub fn get_channel(&self) -> Option<IoChannel> {
         let ch = unsafe { spdk_bdev_get_io_channel(self.0) };
         if ch.is_null() {
+            error!(
+                "failed to get IO channel for, probably low on memory! {}",
```

Comment:
I think the curly braces were meant to be after the "for" i.e. "failed to get IO channel for{},"